src/Stripe.net/Services/Discounts/DiscountService.cs

```diff
@@ -16,24 +16,26 @@ public DiscountService(string apiKey)
         {
         }
 
+        public override string BasePath => null;
```

Comment:
Oh well I guess this is the example of a resource nested under different parents?